thrust/benchmarks/bench/transform/basic.cu

```diff
@@ -33,6 +33,9 @@
 #include <thrust/transform.h>
 #include <thrust/zip_function.h>
 
+#include <cuda/__functional/address_stability.h>
+#include <cuda/functional>
```

Comment:
This broke the tests, since the benchmark is also compiled for the omp, tbb and cpp backends. Including `cuda/functional` also includes headers that require a CUDA compiler.